package-lock.json

```diff
@@ -5954,7 +5034,7 @@
     },
     "onetime": {
       "version": "1.1.0",
-      "resolved": "https://registry.npmjs.org/onetime/-/onetime-1.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/onetime/-/onetime-1.1.0.tgz",
```

Comment:
No idea where this change comes from.

Reply:
I'm investigating

Reply:
Can you try to run again `npm i` in your branch to see if package-lock.json changes?

Reply:
nope, doesn't change

Reply:
It works, but `npm i` reverts all your changes to `package-lock.json`, eg:
```diff
-            "find-up": "^1.0.0",
-            "read-pkg": "^1.0.0"
+            "find-up": "1.1.2",
+            "read-pkg": "1.1.0"
```
I guess it's expected since I don't use npm 6...